MapboxDirections/MBRouteOptions.swift

```diff
@@ -144,8 +110,8 @@ open class RouteOptions: NSObject {
         assert(waypoints.count <= 25, "A route may not have more than 25 waypoints.")
         
         self.waypoints = waypoints
-        self.profileIdentifier = profileIdentifier ?? MBDirectionsProfileIdentifierAutomobile
-        self.allowsUTurnAtWaypoint = ![MBDirectionsProfileIdentifierAutomobile, MBDirectionsProfileIdentifierAutomobileAvoidingTraffic].contains(self.profileIdentifier)
+        self.profileIdentifier = profileIdentifier ?? MBDirectionsProfileIdentifier.automobile.rawValue
+        self.allowsUTurnAtWaypoint = ![MBDirectionsProfileIdentifier.automobile.rawValue, MBDirectionsProfileIdentifier.automobileAvoidingTraffic.rawValue].contains(self.profileIdentifier)
```

Comment:
It's a breaking change and it involves quite a few changes but it would make things clearer.